docs/content/en/docs-dev/user-guide/event-watcher.md

```diff
@@ -165,6 +165,38 @@ pipectl event register \
 
 Note that it is considered a match only when labels are an exact match.
 
+### [optional] Using contexts
+
+You can also attach additional metadata to the event.
+This information can be added as a trailer to the git commit when Event Watcher using the GIT_UPDATE handler.
+This can be useful when attaching information from the source code repository to the manifest repository.
+
+For example, you can attach the source code commit link to the manifest repository.
+
+```bash
+pipectl event register \
+    --address=CONTROL_PLANE_API_ADDRESS \
+    --api-key=API_KEY \
+    --name=sample \
+    --data=gcr.io/pipecd/helloworld:v0.48.0 \
+    --contexts Source-Commit-Hash=xxxxxxx, Source-Commit-URL=https://github.com/pipe-cd/pipecd/commit/xxxxxxx
+```
+
+```bash
+# For manifest repository
+% git show                                                                        
```

Comment:
[nit] you mean this?
```suggestion
# In manifest repository
$ git show                                                                        
```

Reply:
Fixed [09c9cc2](https://github.com/pipe-cd/pipecd/pull/5299/commits/09c9cc20498fa6a542080cd8eb6aa12c81b71bd5)
Thanks, '%' is my terminal prompt. changed to the '$'